app/views/posts/show.html.erb

```diff
@@ -1,4 +1,24 @@
-<h2><%= @post.title %></h2>
-<p><%= @post.text %></p>
-<p>Comments: <%= @post.comments_counter %></p>
-<p>Likes: <%= @post.likes_counter %></p>
+<div class="post-details">
+  <h2 class="mb-0"><%= @post.title %></h2>
+  <p class="text-muted mb-2">Posted by <strong><%= @post.author.name %></strong></p>
+  <p class="text-muted">
+    <span class="me-3">comment count: <strong><%= @post.comments_counter %></strong></span>
+    like count: <strong><%= @post.likes_counter %></strong>
+  </p>

```

Comment:
- [ ] Note: in the `app/views/posts/index.html.erb` file, you're expected to uses `post.recent_comments` to render 5 most recent comments. This method is also missing as its part of the previous [milestone](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md)

### Sample of expected code:
```
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```